go.mod

```diff
@@ -123,6 +123,7 @@ replace (
 	github.com/DataDog/datadog-agent/pkg/util/filesystem => ./pkg/util/filesystem
 	github.com/DataDog/datadog-agent/pkg/util/flavor => ./pkg/util/flavor
 	github.com/DataDog/datadog-agent/pkg/util/fxutil => ./pkg/util/fxutil/
+	github.com/DataDog/datadog-agent/pkg/util/gpu => ./pkg/util/gpu
```

Comment:
I'm not sure this is needed since the gpu package is not a golang module (doesn't have a dedicated `go.mod`)